<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SifremiUnuttum</class>
 <widget class="QMainWindow" name="SifremiUnuttum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Şifremi Unuttum</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>E-Postanızı Giriniz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>E-Postanıza Gelen Kodu Giriniz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Yeni Şifrenizi Giriniz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Yeni Şifrenizi Yeniden Giriniz:</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>139</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="epostaLne"/>
     </item>
     <item>
      <widget class="QPushButton" name="kodYollaBtn">
       <property name="text">
        <string>Kod Yolla</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="kodLne"/>
     </item>
     <item>
      <widget class="QLineEdit" name="yeniSifreLne"/>
     </item>
     <item>
      <widget class="QLineEdit" name="yeniSifreLne2"/>
     </item>
     <item>
      <widget class="QPushButton" name="sifremiDegistirBtn">
       <property name="text">
        <string>Şifreni Değiştir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
